lasagne/tests/test_updates.py

```diff
@@ -65,6 +65,53 @@ def test_updates(self, method, kwargs):
         assert np.allclose(A.get_value(), B.get_value())
         assert np.allclose(A.get_value(), self.torch_values[method])
 
+    @pytest.mark.parametrize('method, kwargs', [
+        ['sgd', {'learning_rate': 0.1}],
+        ['sgd', {'learning_rate': np.float32(0.1)}],
```

Comment:
1. Instead of listing everything twice, can you run it once with floats and then with np.float32 in the test function body?
2. Instead of listing all the arguments, would it be possible to just list the methods and then in the function body, use `inspect.getargspec` to get the default arguments and once call with the defaults and then with the defaults converted to np.float32? This would make it a lot easier to extend if more functions are added.